Microsoft.Toolkit.Uwp.UI/Converters/HexStringToSolidColorBrushConverter

```diff
@@ -0,0 +1,86 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Windows.UI;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Converters
+{
+	/// <summary>
+	///     Convert a hexadecimal color string to a corresponding brush
+	/// </summary>
+	public class HexStringToSolidColorBrushConverter : IValueConverter
```

Comment:
There is something wrong. Your file should be `HexStringToSolidColorBrushConverter.cs`. It lacks the extension.
And by the way, the .csproj does not seem to be updated so your file is outside of the project for the moment.